src/services/StellarContractService.ts

```
@@ -0,0 +1,261 @@
+      logger.error("Payment processing failed:", error);
+      if (error instanceof AppError) {
+        throw error;
+      }
+      throw new AppError("Failed to process payment", 500);
+    }
+  }
+
+  /**
+   * Get merchant details including supported tokens
+   */
+  async getMerchantDetails(merchantAddress: string): Promise<any> {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)